kairon/shared/pos/processor.py

```
@@ -539,6 +540,18 @@ def jsonrpc_get_uid(self, session_id: str) -> int:
         )[0]
 
+    async def send_notification(self, data, bot: str):
+        try:
+            async with httpx.AsyncClient() as client:
+                resp = await client.post(
+                    f"{self.__live_agent_url}/internal/notify/{bot}",
+                    json=data,
+                    timeout=2
+                )
+                return resp.json()
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)